pkg/route/graph/analysis/analysis.go

```diff
@@ -52,6 +56,7 @@ route:
 					Key:      MissingServiceWarning,
 					Message: fmt.Sprintf("%s is supposed to route traffic to %s but %s doesn't exist.",
 						f.ResourceName(routeNode), f.ResourceName(svcNode), f.ResourceName(svcNode)),
+					// TODO: Suggest 'oc create service' once that's a thing.
```

Comment:
There is https://github.com/kubernetes/kubernetes/pull/19509 upstream which hopefully will turn into what we want.